projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -402,6 +401,18 @@ export class IgxLinearProgressBarComponent extends BaseProgressDirective impleme
     constructor() {
         super();
     }
+    public set valueInPercent(value: number) {
+        const val = toValue(value, this._max);
+        if (value < 100) {
```

Comment:
This check value < 100 can be safely removed. If `value` exceeds 100, then `val` will be such a value, that will not pass the check inside the `set value`.